core/controllers/skill_editor.py

```diff
@@ -93,17 +93,14 @@ class SkillRightsHandler(base.BaseHandler):
 
     @acl_decorators.can_edit_skill
     def get(self, skill_id):
-        """Returns the SkillRights object of a skill."""
+        """Returns whether the user can edit the description of a skill."""
```

Comment:
Do we still need the skil_id param online 95? (Considering editing skill rights doesn't depend on skill_id.)

Reply:
Also, I'm not sure whether we should have a separate handler for this or we should add 'can_edit_skill_description` to UserInfoHandler?

Reply:
The name of this handler is SkillRightsHandler, is that expected?